app/services/tree-component-service.ts

```diff
@@ -0,0 +1,14 @@
+import { IActionMapping } from "angular-tree-component";
```

Comment:
Not sure this should really be in the service folder as a service. what is it used for?